_transformations/scale/readme.md

```diff
@@ -14,17 +14,44 @@ install: auto
 
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges.
+The input string must be in numerical format.
 
-The file is expected to exist in the `transform` directory, and should follow the format given in the example below.
+The file is expected to exist in the `transform` directory, and should follow the format given in the table below.
+
+| scale expression | returns XYZ when the given value is |
+|--------------------|--------|
+| `[12..23]=XYZ` | `between (or equal to) 12 and 23` |
+| `]12..23[=XYZ` | `between 12 and 23 (12 and 23 are excluded in this case.)` |
+| `..23]=XYZ` | `lower than or equal to 23` |
+| `]12..=XYZ` | `greater than 12` |
 
 ## Example
 
-transform/temps.scale:
+The following example shows how to break down numeric UV values into fixed UV index categories.
+
+example item:
+
+```java
+Number Uv_Sensor_Level "UV Level [SCALE(uvindex.scale):%s]"
+```
+
+uvindex.scale in transform folder:
 
-```properties
-[12..23]=Temp between 12 or higher and 23 or lower
-]12..23[=Temp over 12 and below 23
-..23]=Any temp up to and including 23
-]12..=Any temp above 12
+```python
+..3]=1
+]3..6]=2
+]6..8]=3
+]8..10]=4
+[10..100]=5
 ```
+
+Each value the item receives, will be categorized to one of the five given ranges.
+Values **lower than or equal to 3** are catched with `..3]=1`.
+Greater values are catched in ranges with 2 values as criteria.
+The only condition here is that the received value has to be lower than or equal to `100` in our example, since we haven't defined other cases yet.
```

Comment:
What happens if none of the scale conditions match? What will be shown (on a sitemap for example), if the received value would be 101?